modules/exploits/windows/iis/iis_webdav_scstoragepathfromurl.rb

```diff
@@ -0,0 +1,175 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ManualRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => ' Microsoft IIS WebDav ScStoragePathFromUrl Overflow',
+      'Description'    => %q{
+          Buffer overflow in the ScStoragePathFromUrl function
+          in the WebDAV service in Internet Information Services (IIS) 6.0
+          in Microsoft Windows Server 2003 R2 allows remote attackers to
+          execute arbitrary code via a long header beginning with
+          "If: <http://" in a PROPFIND request, as exploited in the
+          wild in July or August 2016.
+
+          Original exploit by Zhiniang Peng and Chen Wu.
+      },
+      'Author'         =>
+        [
+        'Zhiniang Peng', # Original author
+        'Chen Wu',       # Original author
+        'Dominic Chell <dominic@mdsec.co.uk>', # metasploit module
+        'firefart', # metasploit module
+        'zcgonvh <zcgonvh@qq.com>' # metasploit module
+        ],
```

Comment:
@rwhitcroft found it on the other PR and added it. If you want smth. else, just ping me